tools/util/profiles/cqlstress_blob_profile.yaml

```diff
@@ -0,0 +1,19 @@
+table: test_table
+
+table_definition:
+  CREATE TABLE IF NOT EXISTS test_table (
+    pkey blob PRIMARY KEY,
+    c1 blob,
+    c2 blob,
+    c3 blob
+   );
+
+keyspace: cqlstress_blob_keyspace
+
+keyspace_definition: |
+  CREATE KEYSPACE IF NOT EXISTS cqlstress_blob_keyspace 
+  WITH replication = { 'class': 'SimpleStrategy', 'replication_factor': 1};
+
+queries:
+  test_query:
+    cql: insert into cqlstress_blob_keyspace.test_table (pkey, c1, c2, c3) values (?, ?, ?, ?)
```

Comment:
Please stick to one case convention in CQL = use CAPITAL LETTERS for CQL keywords.